src/components/QRCodeButton.jsx

```diff
@@ -0,0 +1,59 @@
+// import SwipeableViews from "react-swipeable-views";
+import { makeStyles } from "@material-ui/core/styles";
+import Fab from "@material-ui/core/Fab";
+import { ImQrcode } from "react-icons/im";
+import { Typography } from "@material-ui/core";
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    display: "flex",
+    alignItems: "center",
+    position: "fixed",
+    top: 140,
+    right: theme.spacing(2),
+    zIndex: 2,
+  },
+  labels: {
+    display: "flex",
+    flexDirection: "column",
+    marginRight: theme.spacing(1),
+  },
+  title: {
+    marginBottom: theme.spacing(-1),
+  },
+  subtitle: {
+    letterSpacing: 0.6,
+  },
+  icon: {
+    fontSize: "1.5rem",
+  },
+}));
+
+function QRCodeButton({ showText }) {
+  const classes = useStyles();
+
+  return (
+    <div className={classes.root}>
+      {showText && (
+        <div className={classes.labels}>
+          <Typography component="h3" variant="h6" color="primary" className={classes.title}>
```

Comment:
Siehe Kommentar oben :)
```suggestion
          <Typography component="h3" variant="h6" className={classes.title}>
```